pkg/controller/backup_controller.go

```diff
@@ -426,22 +425,12 @@ func (c *backupController) completeBackupWithUploadedSnapshots(curBackup *pkgbac
 		updateBackupStatusForSnapshotState(curBackup, log)
 	}
 
+	c.updateMetrics(curBackup.Status.Phase, backupScheduleName)
```

Comment:
May need to move updateMetrics after patchBackup function, because patchBackup could fail after metrics are updated.